### DISCUSSION TITLE
My Vercel Postgres Database is not accepting connections

### DISCUSSION THREAD

**Discussion by Manuel-Antunes** [vercel/community#4265] [Help]
### Question

A few days ago, I created a deployment at 02:00 am (BRT), and in the first step of the deployment the application executes a database migration, when the migration was running the Vercel system went down, after the system restarted I noticed a weird behavior, my migration still was running, I supposed that was a bug or something like that, then I stopped the deployment and restarted, and the behavior still was there, in the third try, I got the error '[08P01] FATAL: server login has been failing, try again later (server_login_retry) An I/O error occurred while sending to the backend.' them I noticed that my database was deadlocked and all the available connections were locked, Then I get totally desperate because my system was online but the database was deadlocked without any chance to restart cause there's no way to restart the database connections from Vercel, I tried to contact the support, but no answers until now, and my database usage is beeng computed but I'm not using the database since 09-23-2023 at 2:00 AM because I can't even connect to it, and i will not pay for this issue with your infrastructure, and probable i'll migrate my entire system to AWS just when i can get the access of my data, but giving a little tip, ADD A BIG "RESTART" BUTTON in the database console to avoid some errors like this again.

**Comment by amyegan** [comment]
Hi @Manuel-Antunes. If you let me know the case number, I can check on it for you. 

It will be helpful to know the database id and the team or user id. Please share them in your support case if you haven't already 🙏

**Reply by Manuel-Antunes** [reply]
Here

00160854

**Reply by Manuel-Antunes** [reply]
The organization is fruitpixapp-gmailcom-s-team and i shared the database name in the case

**Reply by amyegan** [reply]
Thanks for doing that. The Storage team restarted things so your database should no longer be stuck. Please let me know if you continue to have trouble with it 🙏

**Reply by Manuel-Antunes** [reply]
I need a new restart in my database, but now in my fallback database, plase check it out!!!

**Reply by Manuel-Antunes** [reply]
![image](https://github.com/vercel/community/assets/57446204/c5e13426-4bac-468f-af7d-21c5f06047f0)
Same error here!!! i need a new restart now!!!

**Reply by Manuel-Antunes** [reply]
Everytime that i deploy a new functionality tonight i got this kind of error.

**Reply by amyegan** [reply]
Are you still getting this error?
